controllers/imagepolicy_controller.go

```diff
@@ -180,6 +180,13 @@ func (r *ImagePolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		var tags []string
 		tags, err = r.Database.Tags(repo.Status.CanonicalImageName)
 		if err == nil {
+			if len(tags) == 0 {
+				msg := fmt.Sprintf("no tags found in '%s', retrying in %s", repo.Name, repo.Spec.Interval.Duration)
+				r.event(ctx, pol, events.EventSeverityInfo, msg)
```

Comment:
Please log this as well, issuing events only is not something we should do IMO.

Reply:
Added the logs and will generate another rc image with it.

Reply:
Thanks